lib/jsonapi/rails/renderer.rb

```diff
@@ -33,6 +33,7 @@ def rails_renderer(renderer)
         reverse_mapping = request.env[ActionController::REVERSE_MAPPING_KEY]
         options = options.merge(_reverse_mapping: reverse_mapping)
         json = renderer.render(json, options) unless json.is_a?(String)
+        json = json.to_json unless json.is_a?(String)
```

Comment:
Won't this break code which is pinned to the previous version of `jsonapi-serializable` (before `.to_json` was removed)? If `renderer.render` returns a string, calling `.to_json` on it will result in additional round of escaping.